app/post_metadata_test.go

```diff
@@ -189,7 +190,13 @@ func TestPreparePostForClient(t *testing.T) {
 
 		fileInfo.PostId = post.Id
 
-		clientPost := th.App.PreparePostForClient(post, false, false)
+		var clientPost *model.Post
+		assert.Eventually(t, func() bool {
+			p, nErr := th.App.Srv().Store.Post().Get(context.Background(), post.Id, false, false, false, "")
+			require.NoError(t, nErr)
```

Comment:
Wondering why do we need to do this. Isn't calling `PreparePostForClient` repeatedly sufficient?

Reply:
Yes, probably we don't need it, good catch.

Reply:
Done